setup.py

```diff
@@ -1,8 +1,13 @@
+from os import sys, path
+
 from skbuild import setup
 
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from itkVersion import VERSION
```

Comment:
> assumed that setuptools automatically populated <package>.__version__

That was a reasonable assumption.

But since it is not possible to declare a package as the "main" one, it has to be done explicitly.  

For example, here is how it is done for ninja distribution ... https://github.com/scikit-build/ninja-python-distributions/blob/ee84aedab365b811fe61ff2739d10f988ba81c9e/ninja/__init__.py#L5-L8